contrib/storage-phoenix/README.md

```diff
@@ -87,6 +79,16 @@ Tips :
 }
 ```
 
+### Impersonation
+Configurations :
+1. Enable [Drill User Impersonation](https://drill.apache.org/docs/configuring-user-impersonation/)
+2. Enable [PQS Impersonation](https://phoenix.apache.org/server.html#Impersonation)
+3. PQS URL:
+  1. Provide `host` and `port` and Drill will generate the PQS URL with a doAs parameter of current session user
+  2. Provide the `jdbcURL` with a `doAs` url param and `$user` placeholder as a value, for instance:
```

Comment:
Hi Christian,

`doAs=$user` is adding automatically in case the first approach of configuring PQS is chosen - `host` and `port`. As Paul mentioned [earlier](https://github.com/apache/drill/pull/2332#discussion_r758116891) in the original PR #2332 for Phoenix plugin, the second approach is more advanced case and for more manual configuration. So I preferred to avoid automagic change of the final URL.

Reply:
Ah... Sounds legit. Thanks.